zephyr/ra/ra_cfg/fsp_cfg/bsp/ra8m1/bsp_clock_cfg.h

```diff
@@ -62,65 +58,42 @@
+#define BSP_CFG_FCLK_DIV     RA_CGC_CLK_DIV(DT_NODELABEL(fclk), div, 8)
 
-#define BSP_CFG_UCK_SOURCE                                                                         \
-	BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(uclk), clk_src, RA_CLOCK_SOURCE_DISABLE)
-#define BSP_CFG_UCK_DIV BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(uclk), clk_div, 0)
-#define BSP_CFG_U60CK_SOURCE                                                                       \
-	BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(u60clk), clk_src, RA_CLOCK_SOURCE_DISABLE)
-#define BSP_CFG_U60CK_DIV BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(u60clk), clk_div, 0)
-#define BSP_CFG_OCTA_SOURCE                                                                        \
-	BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(octaspiclk), clk_src,                       \
-					  RA_CLOCK_SOURCE_DISABLE)
-#define BSP_CFG_OCTA_DIV BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(octaspiclk), clk_div, 0)
-#define BSP_CFG_CANFDCLK_SOURCE                                                                    \
-	BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(canfdclk), clk_src, RA_CLOCK_SOURCE_DISABLE)
-#define BSP_CFG_CANFDCLK_DIV BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(canfdclk), clk_div, 0)
-#define BSP_CFG_CLKOUT_SOURCE                                                                      \
-	BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(clkout), clk_src, RA_CLOCK_SOURCE_DISABLE)
-#define BSP_CFG_CLKOUT_DIV BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(clkout), clk_div, 0)
-#define BSP_CFG_SCICLK_SOURCE                                                                      \
-	BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(sciclk), clk_src, RA_CLOCK_SOURCE_DISABLE)
-#define BSP_CFG_SCICLK_DIV BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(sciclk), clk_div, 0)
-#define BSP_CFG_SPICLK_SOURCE                                                                      \
-	BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(spiclk), clk_src, RA_CLOCK_SOURCE_DISABLE)
-#define BSP_CFG_SPICLK_DIV BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(spiclk), clk_div, 0)
-#define BSP_CFG_ADCCLK_SOURCE                                                                      \
-	BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(adcclk), clk_src, RA_CLOCK_SOURCE_DISABLE)
-#define BSP_CFG_ADCCLK_DIV BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(adcclk), clk_div, 0)
-#define BSP_CFG_I3CCLK_SOURCE                                                                      \
-	BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(i3cclk), clk_src, RA_CLOCK_SOURCE_DISABLE)
-#define BSP_CFG_I3CCLK_DIV BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(i3cclk), clk_div, 0)
+#define BSP_CFG_UCK_SOURCE      RA_CGC_CLK_SRC(DT_CLOCKS_CTLR(DT_NODELABEL(uclk)))
+#define BSP_CFG_UCK_DIV         RA_CGC_CLK_DIV(DT_NODELABEL(uclk), div, 1)
+#define BSP_CFG_U60CK_SOURCE    RA_CGC_CLK_SRC(DT_CLOCKS_CTLR(DT_NODELABEL(u64clk)))
```

Comment:
@soburi , This seem to be a mistake
```suggestion
#define BSP_CFG_U60CK_SOURCE    RA_CGC_CLK_SRC(DT_CLOCKS_CTLR(DT_NODELABEL(u60clk)))
```

Reply:
Thank you for catching.
I fixed it.
And found some mistakes in ra6m1, ra8d1 and ra8t1, also fixed.